### PR TITLE
Add ability to install JCE policy files for oracle JDK

### PR DESCRIPTION
**Pull request by spion06**

### CODE REVIEW

**Comment by spion06** [comment]
Just wanted to check if there is anything else you need from me on this pull.

Thanks!

File: test/integration/oracle/bats/verify_oracle.bats [hunk 1]
```diff
@@ -12,3 +12,8 @@
   run test -L /usr/bin/jar
   [ "$status" -eq 0 ]
 }
+
+@test "properly installs JCE" {
+  run java -jar /tmp/UnlimitedSupportJCETest.jar
```
Comment (erichelgeson) [inline test/integration/oracle/bats/verify_oracle.bats]: Can you do that as well? I'll merge this.

**Comment by erichelgeson** [comment]
Sorry, busy over holidays and work. I'm trying to cut a release this week.

File: test/integration/oracle/bats/verify_oracle.bats (hunk 1 above)
Comment (spion06) [inline test/integration/oracle/bats/verify_oracle.bats]: Submitted #231 to add the source code into the test cookbook

**Pull request merged**